RFCs/0023-TestSettings-Deprecation.md

```diff
@@ -0,0 +1,31 @@
+# 0023 TestSettings Deprecation
+
+## Summary
+
+This note outlines the proposed changes to deprecate support for testsettings in test runs.
```

Comment:
Should be "to deprecate support for using .testsettings in automated unit and functional testing scenarios."
We should also add that .testsettings itself will continue to be supported for load test scenarios.